src/new-client/src/Web/TearOutSection/handleTearOutSection.web.ts

```diff
@@ -0,0 +1,33 @@
+import { constructUrl } from "@/utils/url"
+import { openWindow } from "@/utils/window/openWindow"
+import { tearOutSection } from "./state"
+
+type Section = "liverates" | "trades" | "analytics"
```

Comment:
Suggest moving this to `TearOutSection/state.ts` and using it everywhere in the app that references the sections (rather than just a string)

Reply:
Done